rollup-interface/src/state_machine/mocks/da.rs

```diff
@@ -302,12 +304,12 @@ impl DaSpec for MockDaSpec {
     type ChainParams = ();
 }
 
-#[cfg(feature = "native")]
+#[cfg(all(feature = "native", feature = "tokio"))]
 use tokio::sync::mpsc::{self, Receiver, Sender};
-#[cfg(feature = "native")]
+#[cfg(all(feature = "native", feature = "tokio"))]
 use tokio::sync::Mutex;
 
-#[cfg(feature = "native")]
+#[cfg(all(feature = "native", feature = "tokio"))]
 #[derive(Clone)]
 /// DaService used in tests.
 pub struct MockDaService {
```

Comment:
It seems that `MockDaService` requires a ton of feature gating. Can we maybe put it (and its required imports) in a module and re-export it with just 2 feature gates?

Reply:
Moved to `src/state_machine/mocks/service.rs`